shared/lib/components.js

```diff
@@ -206,7 +206,7 @@ function renderPreview(componentName, options) {
       footer: '',
 
       main: outdent`
-        <div class="govuk-width-container">
```

Comment:
I wonder if we can link to this function in the `helpers/puppeteer.render` JSDoc when we mention  the `test boilerplate page`. Would've made it a bit easier for this addled brain to figure out where that was coming from.

Reply:
Yeah another good idea. I've added some JSDoc `@link` references